pkg/util/precision/converter.go

```diff
@@ -21,6 +22,7 @@ type (
 	// This is a JSON bound that uses neo node. Neo-go has int64 limit for
 	// `smartcontract.Parameter` of integer type.
 	Fixed8Converter struct {
+		m sync.RWMutex
```

Comment:
Is it really worth the trouble? Why not recreating the converter where needed?

Reply:
I mean using `NewConverter()` when needed (on reinit). Once initialized it never changes. If connection is broken then all users of it terminate anyway. Once it's up again we'll have a new converter and they will use this new one. Something like that (I'm not entirely sure it fits the current code well, but it theory it should).

Reply:
> Once initialized it never changes.

oh, you mean that we can not take into account that [precision](https://github.com/notimetoname/neofs-node/blob/0aee98732d75f67446df2167a294e560bad74039/pkg/util/precision/converter.go#L76) may change while we were down? or i got you wrong one more time?

Reply:
made the precision constant

Reply:
I take as [NEP-17 `decimals()`](https://github.com/neo-project/proposals/blob/06a6b550987a74025009cf6841757f2426462697/nep-17.mediawiki#decimals). It can't change for a contract. And it's the same here, no one should even of changing it on a real network. It can only happen if you're to create another contract, another token and migrate to it. But this is a completely different story.